### PR TITLE
feat(docs): FRMW-195 - Migrate rxSessionStorage docs

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-195

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

File: src/rxSessionStorage/README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
 [![unstable](http://badges.github.io/stability-badges/dist/unstable.svg)](http://github.com/badges/stability-badges)
 
-A simple wrapper of the global sessionStorage object for interacting with session storage.
-This service is similar to angular's $window and $document services.  The API is exactly the
-same as the W3C's specification provided at: http://dev.w3.org/html5/webstorage/#storage-0
+A component that creates a simple wrapper for injecting the global variable
+sessionStorage for storing values in session storage.
```
Comment (CITguy) [inline src/rxSessionStorage/README.md]: I'd go with:

```
A component that provides a simple wrapper around the global `sessionStorage` object for interacting with session storage.
```

Comment (bry4n) [inline src/rxSessionStorage/README.md]: 10-4

**Comment by CITguy** [comment]
LGTM

**Comment by 100stacks** [comment]
LGTM

**Pull request merged**